### ISSUE TITLE
.dataAlpha as last line consumes last character

### ISSUE THREAD

**Issue by kommisar5150** [CommunicationsSecurityEstablishment/spartacus#31]
If we have a .dataAlpha field as the last line of a .casm file with **no new line** at the end, the assembler
consumes the last character and replaces it with null. Usually, the new line character would be consumed instead. This is a special case that would need separate handling. Either we enforce coders to add a new line at the end, or we change the .dataAlpha code field a little.

**Comment by pg-cse** [comment]
Good catch! This bug was in the code since V1.0. Kudos for catching this!
